backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -26,16 +25,17 @@ module.exports = class CrmPage {
    * @param {String} selector - The selector that identifies the select2 dropdown
    * @param {String} label
    */
-  async clickSelect2Option(selector, label) {
+  async clickSelect2Option (selector, label) {
     await this.engine.click(`${selector} [class^="select2-choice"]`);
     await this.engine.evaluate(label => {
       const xPath = `.//div[contains(@class, "select2-result-label")][text()="${label}"]/parent::*`;
+      let XPathResult;
```

Comment:
A definition of an empty variable, then next line you use it as an object?
Could you please explain what is it for?

Reply:
I have added this line as semi-standard was throwing error because XPathResult is used in the next line but not defined.

Reply:
This is wrong and breaks the code. It is just a global variable (which is not good, but it is what it is). You should put in the top of the file:

```js
/* globals XPathResult */
```

Reply:
Done!